docs/en/observability/profiling-self-managed.asciidoc

```diff
@@ -0,0 +1,840 @@
+[[profiling-self-managed]]
+= Run Universal Profiling on self-hosted Elastic stack
+
+++++
+<titleabbrev>Self-hosted infrastructure</titleabbrev>
+++++
+
+beta::[]
+
+Here you'll find information on running Universal Profiling when hosting the Elastic stack on your own infrastructure.
+
+Universal Profiling consists of a client part (the host-agent) and a server part (the backend). This documentation focuses on the server part.
+
+[discrete]
+[[profiling-self-managed-supported-platforms]]
+== Prerequisites
+
+IMPORTANT: Running Universal Profiling on self hosted infrastructure requires an Elastic **Enterprise license**.
+
+* Elastic stack: minimum version 8.12.0, on any Linux distribution (x86_64 or ARM64 architectures), with a Kernel 4.x or higher.
+* https://www.elastic.co/ece[ECE]: minimum version 3.7.0, using the 8.12.0 stackpack or higher.
+* Kubernetes: version 1.22+, using Helm charts.
+
+NOTE: For Elastic Cloud on Kubernetes (ECK), the Universal Profiling backend can be installed using helm charts in standalone mode.
+Once installed, you can connect the profiling backend to an existing ECK-managed stack.
+Currently, we do not have Profiling Operators and the CRDs necessary for running the Profiling backend natively in ECK.
+
+[discrete]
+[[profiling-self-managed-backend-support-matrix]]
+=== Supported platforms
+
+The following platforms were tested and successfully ran the Universal Profiling backend.
+
+.Expand to see platform details.
+[%collapsible]
```

Comment:
I'm not sure a collapsible widget does much here. What's the use case for hiding this information? Won't everyone on this page need it?

Reply:
I thought since it's a big table it would hinder page readability for someone who's already visited the page.
If you think it should not be collapsible, I am ok removing it.

Reply:
I've updated to remove it. I think the info is important, and the table shows the it in a nice way that will be noticeable when scanning. When collapsed, users scanning the doc would probably miss it.